src/TBuffer.cpp

```diff
@@ -2253,6 +2253,64 @@ void TBuffer::append(const QString& text, int sub_start, int sub_end, TChar form
     }
 }
 
+// Wraps text to max line length of mWrapAt
+// Applies indentation of mWrapIndent to wrapped lines
+QString TBuffer::wrapText(const QString& text) const
+{
+    if (mWrapAt <= mWrapIndent) {
+        qWarning() << "mWrapAt (" << mWrapAt << ") is too small to accommodate mWrapIndent (" << mWrapIndent << ")";
+    }
+
+    const QString wordBreaks = qsl(",.- ");
```

Comment:
This methodology, (of looking for these particular characters in the text and considering splitting the line where they are is ***fundamentally*** flawed because it cannot work for locales that do NOT use any of these characters (including spaces) between textual "units" {e.g. Chinese, either Traditional OR Simplified, see https://chinese.stackexchange.com/q/17763 }.

I have long held the opinion that Mudlet needs to instead look at using the `QTextBoundaryFinder` and finding the point in the text where `QTextBoundaryFinder::Line` type boundaries are...